File: packages/cli/test/fixtures/ranking-corpora/java/src/main/java/com/example/ranking/util/Logger.java
package com.example.ranking.util;

/**
 * Shared logging helper -- the hub class every handler in this fixture
 * corpus imports.
 */
public class Logger {
    public static void logInfo(String message) {
        System.out.println("[info] " + message);
    }

    public static void logError(String context, Exception err) {
        System.out.println("[error] " + context + ": " + err.getMessage());
    }
}

File: packages/cli/test/fixtures/ranking-corpora/kotlin/src/main/kotlin/com/example/ranking/util/Logger.kt
package com.example.ranking.util

/**
 * Shared logging helper -- the hub object every handler in this fixture
 * corpus imports.
 */
object Logger {
    fun logInfo(message: String) {
        println("[info] $message")
    }

    fun logError(context: String, err: Exception) {
        println("[error] $context: ${err.message}")
    }
}
